lib/normalize_title_filter.js

```diff
@@ -29,6 +29,7 @@ module.exports = (hyper, req, next, options, specInfo) => {
             }
         });
     }
+    rp.title = (options.title && rp[options.title]) || rp.title;
```

Comment:
Our overall goal here is to get rid of the hard-coded property name `title` right? So why are we assigning the request parameter `title`? We know that that the page name that we need to normalize will be located inside the `options.title` property right? 

Also, having the options property named title is a bit confusing isn't it? now we have so many titles in the code that it's hard to understand which one refers to what. Would it be more understandable if the options property be called 'titleParamName' here as well as in the utility function?